torchgeo/datasets/idtrees.py

```diff
@@ -549,10 +550,10 @@ def plot_las(self, index: int, colormap: Optional[str] = None) -> Any:
             else:
                 colors = np.zeros_like(points)
 
-        pcd = open3d.geometry.PointCloud()
-        pcd.points = open3d.utility.Vector3dVector(points)
-        pcd.colors = open3d.utility.Vector3dVector(colors)
-        vis = open3d.visualization.Visualizer()
-        vis.create_window()
-        vis.add_geometry(pcd)
-        return vis
+        point_cloud = pyvista.PolyData(points)  # type: ignore[attr-defined]
+        point_cloud["color"] = colors
+        plotter = pyvista.Plotter()  # type: ignore[attr-defined]
+        _ = plotter.add_mesh(point_cloud, scalars="color", rgb=True)
+        plotter.camera_position = "yz"
+        plotter.background_color = "white"
```

Comment:
Users can still modify the plotter object. This simply plots the interactive colored point cloud on a white background.

Reply:
I recommend not setting this background and letting PyVista's theme determine the background. Users can easily control this with `pv.set_plot_theme('document')` (before creating a plotter). We may change the default PyVista theme soon here as grey backgrounds aren't the best IMO